`TestE2EMefEline::test_010_patch_inter_evc` and `test_020_patch_intra_evc` are now failing

`available_tags` looks coherent, looks like a logical assertion error, @Ktmi can you fix these two test cases?

```
tests/test_e2e_18_mef_eline.py:72: AssertionError
tests/test_e2e_18_mef_eline.py::TestE2EMefEline::test_010_patch_inter_evc: 2025-09-25,06:06:25.424509 - 2025-09-25,06:06:25.994532
tests/test_e2e_18_mef_eline.py::TestE2EMefEline::test_020_patch_intra_evc: 2025-09-25,06:07:45.115144 - 2025-09-25,06:07:46.131830
=========================== rerun test summary info ============================
RERUN tests/test_e2e_18_mef_eline.py::TestE2EMefEline::test_010_patch_inter_evc
RERUN tests/test_e2e_18_mef_eline.py::TestE2EMefEline::test_010_patch_inter_evc
RERUN tests/test_e2e_18_mef_eline.py::TestE2EMefEline::test_020_patch_intra_evc
RERUN tests/test_e2e_18_mef_eline.py::TestE2EMefEline::test_020_patch_intra_evc
=========================== short test summary info ============================
FAILED tests/test_e2e_18_mef_eline.py::TestE2EMefEline::test_010_patch_inter_evc - AssertionError: [[1, 99], [101, 3798], [3800, 4094]]
assert False
FAILED tests/test_e2e_18_mef_eline.py::TestE2EMefEline::test_020_patch_intra_evc - AssertionError: [[1, 99], [101, 3798], [3800, 4094]]
assert False
```


```
>       self.assert_tag_used_by_interface(
            "00:00:00:00:00:00:00:01:2",
            "vlan",
            100
        )

tests/test_e2e_18_mef_eline.py:220:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tests.test_e2e_18_mef_eline.TestE2EMefEline object at 0x7f47676dbc50>
interface_id = '00:00:00:00:00:00:00:01:2', tag_type = 'vlan', value = 100

    def assert_tag_used_by_interface(self, interface_id, tag_type, value):
        api_url = KYTOS_API + "/kytos/topology/v3/interfaces/tag_ranges"
        response = requests.get(api_url)
        assert response.status_code == 200, response.text

        data = response.json()

        interface_tags = data[interface_id]
        available_tags = interface_tags["available_tags"][tag_type]
        # tag_ranges = interface_tags["tag_ranges"][tag_type]

        for range_start, range_end in available_tags:
            if range_start <= value <= range_end:
                break
        else:
            assert True, available_tags
>       assert False, available_tags
E       AssertionError: [[1, 99], [101, 3798], [3800, 4094]]
E       assert False
```

--- a/tests/test_e2e_18_mef_eline.py
+++ b/tests/test_e2e_18_mef_eline.py
@@ -35,24 +35,6 @@ class TestE2EMefEline:
                 "up"
             )
 
-    def is_tag_used_by_interface(self, interface_id, tag_type, value):
-        api_url = KYTOS_API + "/kytos/topology/v3/interfaces/tag_ranges"
-        response = requests.get(api_url)
-        assert response.status_code == 200, response.text
-
-        data = response.json()
-
-        interface_tags = data[interface_id]
-        available_tags = interface_tags["available_tags"][tag_type]
-        # tag_ranges = interface_tags["tag_ranges"][tag_type]
-
-        for range_start, range_end in available_tags:
-            if range_start <= value <= range_end:
-                break
-        else:
-            return True
-        return False
-    
     def assert_tag_used_by_interface(self, interface_id, tag_type, value):
         api_url = KYTOS_API + "/kytos/topology/v3/interfaces/tag_ranges"
         response = requests.get(api_url)

--- a/tests/test_e2e_18_mef_eline.py
+++ b/tests/test_e2e_18_mef_eline.py
@@ -69,6 +69,7 @@ class TestE2EMefEline:
                 break
         else:
             assert True, available_tags
+            return
         assert False, available_tags
 
     def assert_tag_not_used_by_interface(self, interface_id, tag_type, value):

